packages/phone-number-privacy/signer/src/salt-generation/get-salt.ts

```diff
@@ -52,7 +52,9 @@ export async function handleGetBlindedMessageForSalt(
     // In the case of a DB or blockchain connection failure, don't block user
     // but set the error status accordingly
     try {
-      ;[performedQueryCount, totalQuota] = await getRemainingQueryCount(account, hashedPhoneNumber)
+      const queryCount = await getRemainingQueryCount(account, hashedPhoneNumber)
+      performedQueryCount = queryCount.performedQueryCount
+      totalQuota = queryCount.totalQuota
     } catch (error) {
```

Comment:
i think you can set it similar to how you had it before `const { performedQueryCount, totalQuota } = await getRemainingQueryCount(account, hashedPhoneNumber)`

like https://github.com/celo-org/celo-monorepo/blob/amyslawson/combiner_second_region/packages/phone-number-privacy/signer/src/salt-generation/get-salt.ts#L38

Reply:
Just tested it out.. If I do that, I'll still need to assign them to the variables defined before the try clause. This assignment treats them as new variables instead of assigning the value to the previously defined ones. So in the end, I'll need the same number of lines for the assignment.